.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "root": true,
+    "env": {
+        "browser": true,
+        "node": true,
+        "cypress/globals": true
+    },
+    "extends": ["airbnb", "eslint:recommended", "plugin:cypress/recommended"],
```

Comment:
Is this necessary?

Reply:
Yes, this is necessary to make Prettier and Eslint work as a team; with this configuration, you could get the best of both worlds.

In this case, they use the following order:
- Apply Cypress code style guidelines, just for Cypress stuff.
- Apply [AirBnB Javascript code style guidelines](https://medium.com/@corentinjacques/airbnbs-javascript-style-guide-5fe9100d2386) for the rest of the Javascript code.
- Apply Eslint's recommendation for those the previous ones didn't have a rule.
- Default: apply Prettier code style guidelines.